packages/react/src/components/TextFieldBase/TextFieldBase.styles.ts

```diff
@@ -22,16 +22,20 @@ export const useStyles = css({
     fontSize: '$x-large',
     justifyContent: 'center',
     padding: '$small',
-    pointerEvents: 'none',
+    pointerEvents: 'auto',
     position: 'absolute',
     top: 0,
     zIndex: 2,
-
+    bottom: 0,
     '> .material-icons': {
       fontSize: '$x-large',
     },
   },
 
+  '.manifest-textfield-base__icon_allow_pointer_events': {
```

Comment:
look like this one is not used anywhere

Reply:
Thanks @vitaliy-p44 for pointing out, removed this.